src/hooks/use-broadcast-channel.ts

```diff
@@ -17,19 +17,16 @@ export default function useBroadcastChannel(
   onMessage: MessageCallback = noop,
   deps = []
 ) {
-  const [broadcasterId] = useState(() => ulid());
-  const [channel, setChannel] = useState(
-    () => new BroadcastChannel(channelName)
-  );
+  const broadcasterId = useRef(() => ulid());
```

Comment:
Do we need `ulid` with `useId()` in React 18?

Reply:
We have a React peer dep of `>= 17.0.2` (or something like that), so I think using a React-18 specific API would break that, no?

Reply:
Ah yes, not worth adjusting that here.